qml/LogicalView.qml

```diff
@@ -63,6 +63,10 @@ Rectangle {
             property bool highlight: false
             property var topicList: topicList
 
+            ListView.onAdd: {
```

Comment:
The following error appears when init a monitor:
`qrc:/qml/LogicalView.qml:206: TypeError: Value is null and could not be converted to an object`